### ISSUE TITLE
Problem with reminders

### ISSUE THREAD

**Issue by Kateylynn4** [quarck/CalendarNotification#314]
I want reminders 15 minutes before, 10 minutes before, 5 minutes before, 1 minute before, and 0 minutes (or as close as I can get to it) a meeting starts. I tried using reminders but when I want the reminder to go away for now, I snooze the reminder by swiping it, and it shows a reminder 5 minutes later regardless of whether that matches up with my set reminders. So let's say I snooze a reminder 3 minutes before a meeting. I should get my next reminder 1 minute before. Instead the app reminds me 2 minutes after the meeting started because that is 5 minutes after I last snoozed.

I tried setting up these intervals in snooze but that didn't work either.

**Comment by quarck** [comment]
> So let's say I snooze a reminder 3 minutes before a meeting

When you snooze, you have to choose the time. Did you choose to remind "1 minutes before" when snoozing?

**Comment by Kateylynn4** [comment]
Ideally I want it to automatically snooze until 1 minute before the meeting. That's why I set up my Reminder Intervals as 15m, 10m, 5m, 1m, 15s. Is that not what the Reminder Intervals are for?

**Comment by quarck** [comment]
Reminder intervals are intervals for reminders to fire. Not "times before the event". So if you set it to " 15m, 10m, 5m, 1m, 15s" it means it would fire first reminder 15 mins after the event, second - another 10m after (25m all together from the initial event), third - another 5m after (30m after the initial), forth - another 1m  after (31m after initial event), fifth - 15s after (or 31m15s after initial event), sixth - another 15m after (yes, it loops over, so total time is 46m15s after the event). And so on... 

What you need, is to add reminders into event itself (using your main calendar app - don't know what exactly you are using) to fire a reminder at 1 minute before, 5 minutes before, 10 minutes before, 15 minutes before, etc. Using the main calendar app, not this one.

**Comment by Kateylynn4** [comment]
I am using my Samsung calendar and it only allows one reminder which is why I downloaded this app.

**Comment by Kateylynn4** [comment]
If I set the initial reminder to 15 minutes using my Samsung calendar, could I use reminder intervals of 5m, 5m, 4m, 45s? That would make the reminders pop up at 10 minutes, 5 minutes, 1 minute, and 15 seconds before my meeting. Right?

**Comment by quarck** [comment]
That's correct, it should work like that. Just make sure to disable any battery optimisations for the app in the android settings and I'll recommend turning on (if not on yet) the option "Use setAlarmClock for reminders" under app settings -> misc. This will ensure reminders arrive on time. (Otherwise android might "optimize" wake times for the app and it would remind at later time). 

But in general if you need app for just that - you could use the built-in samsung calendar functionality. When you are creating a new event you can add as many event reminders as you want using the "add button": 

https://imgur.com/a/GwA6lfE

Also if you'd use Google Calendar instead of Samsung Calendar (you could install it from Play Store and disable notifications inside Samsung Calendar), then you could configure more than one default reminder for new events: 

https://imgur.com/a/bTkPIAh

**Comment by Kateylynn4** [comment]
Hmmm, I dont have that Add button in my Samsung calendar. I am using version 10.0.00.64 on my AT&T Samsung 9+

And I wish I could use Gmail (that's what got me so reliant on the multiple reminders) but when i display my work calendar in that app, it only allows me to view those events, not change them.

**Comment by Kateylynn4** [comment]
Hmmm, I dont have that Add button in my Samsung calendar. I am using version 10.0.00.64 on my AT&T Samsung 9+

And I wish I could use Gmail (that's what got me so reliant on the multiple reminders) but when i display my work calendar in that app, it only allows me to view those events, not change them.

**Comment by Kateylynn4** [comment]
Ok so I have the reminder intervals set for 5m, 5m, 4m, 45s. When the reminder pops up, what do I do so the app respects the next interval? I tried swiping to snooze but that ignores the reminder intervals and just reminds me again in 5 minutes. And I tried dismiss but that stops all reminders. I tried ignoring the reminder and allowing it to stay in the notification area on my screen but that stopped all reminders too.

**Comment by Kateylynn4** [comment]
Ok so I have the reminder intervals set for 5m, 5m, 4m, 45s. When the
reminder pops up, what do I do so the app respects the next interval? I
tried swiping to snooze but that ignores the reminder intervals and just
reminds me again in however many minutes I have the default snooze set to.
And I tried dismiss but that stops all reminders. I tried ignoring the
reminder and allowing it to stay in the notification area on my screen but
that stopped all reminders too.
Sign up for free <https://github.com/join?source=comment-repo>
